### PR TITLE
namespace moves for more transforms

### PR DESCRIPTION
**Pull request by sfilipi**
More changes related to #1318

### CODE REVIEW

**Comment by JRAlexander** [comment]
Don't have perms to review, but LGTM.

File: src/Microsoft.ML.Legacy/CSharpApi.cs [hunk 1]
```diff
@@ -1582,18 +1582,6 @@ public void Add(Microsoft.ML.Legacy.Transforms.Scorer input, Microsoft.ML.Legacy
                 _jsonNodes.Add(Serialize("Transforms.Scorer", input, output));
             }
 
-            public Microsoft.ML.Legacy.Transforms.Segregator.Output Add(Microsoft.ML.Legacy.Transforms.Segregator input)
```
[the remark this answers is not in the document]
Comment (sfilipi) [reply src/Microsoft.ML.Legacy/CSharpApi.cs]: I segregated the Group and Ungroup transform's changes in two different PRs, and this happened. Fixed it in PR6, and fixing it here atm.

File: src/Microsoft.ML.Transforms/WrappedGcnTransformers.cs [hunk 2]
```diff
@@ -7,9 +7,10 @@
 using Microsoft.ML.Runtime.Data;
 using Microsoft.ML.StaticPipe;
 using Microsoft.ML.StaticPipe.Runtime;
+using Microsoft.ML.Transforms.Projections;
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Transforms/WrappedGcnTransformers.cs]: >Projections [](start = 30, length = 11)

What about Pca, whitening?

File: src/Microsoft.ML.Transforms/GcnTransform.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@
 
 [assembly: EntryPointModule(typeof(LpNormalization))]
 
-namespace Microsoft.ML.Runtime.Data
+namespace Microsoft.ML.Transforms.Projections
```
Comment (Ivanidzo4ka) [inline src/Microsoft.ML.Transforms/GcnTransform.cs]: >Projections [](start = 34, length = 11)

PCA, Whitening, LDA at some extent is also projections.

Comment (sfilipi) [reply src/Microsoft.ML.Transforms/GcnTransform.cs]: is LDA projections or text?

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/GcnTransform.cs]: it's projection of text! so Text, (Well it's operate on vector of floats which we get from word bag, but it's main purpose is text related)
I forgot RFF, it's also projection.
Actually Whitening on internal side is stated in Image category. 

---
In reply to: [229512989](https://github.com/dotnet/machinelearning/pull/1453#discussion_r229512989) [](ancestors = 229512989)

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/GcnTransform.cs]: Well, fun fact, GCN is also stated as Image transform :)

---
In reply to: [229514079](https://github.com/dotnet/machinelearning/pull/1453#discussion_r229514079) [](ancestors = 229514079,229512989)

File: test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs [hunk 4]
```diff
@@ -248,7 +248,7 @@ private string GetBuildPrefix()
 #endif
         }
 
-        [Fact(Skip = "Execute this test if you want to regenerate the core_manifest and core_ep_list files")]
```
Comment (sfilipi) [inline test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs]: >(Skip = "Execute this test if you want to regenerate the core_manifest and core_ep_list files") [](start = 13, length = 95)

oopps

File: test/Microsoft.ML.Tests/CSharpCodeGen.cs [hunk 5]
```diff
@@ -15,7 +15,7 @@ public CSharpCodeGen(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact(Skip = "Execute this test if you want to regenerate CSharpApi file")]
```
Comment (sfilipi) [inline test/Microsoft.ML.Tests/CSharpCodeGen.cs]: > [Fact(Skip = "Execute this test if you want to regenerate CSharpApi file")] [](start = 7, length = 76)

revert

**Pull request merged**